engines/dreamweb/stubs.cpp

```diff
@@ -929,7 +930,9 @@ void DreamWebEngine::processTrigger() {
 	}
 }
 
+const char* lastText = "";
```

Comment:
The code formatting here is incorrect.

Reply:
Also, why put this on a global level? Please move it to the engine class.

Reply:
My reasoning to putting it at global level, is because the function is called many times to print the same string, so it needs something outside the scope to detect if TTS should speak. Would you suggest me to pass an external variable to the function instead of declaring a global variable?